index.js

```diff
@@ -22,7 +27,16 @@ module.exports = {
 
   init: function() {
     var bowerDeps = this.project.bowerDependencies();
+
     var VersionChecker = require('ember-cli-version-checker');
+    var options = this.options = this.options || {};
```

Comment:
But it's never used again in this file, that's why I am asking. Is `init` called multiple times?